src/App.css

```diff
@@ -151,6 +163,35 @@ input:-webkit-autofill:active {
   cursor: pointer;
 }
 
+/*--------------------------------------------------------------------------*\
+     LIST VIEW
+\*--------------------------------------------------------------------------*/
```

Comment:
Separating the CSS of the different views makes this code much more readable, good choice to go with comments 👍🏽